packages/gestalt/src/Button.js

```diff
@@ -247,7 +235,7 @@ const ButtonWithForwardRef: React$AbstractComponent<
         onTouchMove={handleTouchMove}
         onTouchStart={handleTouchStart}
         ref={innerRef}
-        style={compressStyle}
+        style={compressStyle || undefined}
```

Comment:
@AlbertCarreras why this change?

Reply:
Because it's null, therefore some snapshots show `style='null'`  
![image](https://user-images.githubusercontent.com/10593890/92270039-2e74c400-ee9a-11ea-90fb-d7a7be101b81.png)